MANIFEST.in

```diff
@@ -1,15 +0,0 @@
-include requirements.*
```

Comment:
Are you sure we are good to delete this now? won't this result in build usues?

Reply:
nah you don't need it, we use `pyproject.toml` now

Reply:
maybe removing the custom builder solves for this?

Reply:
Yeah you don't need it.

Try `uv build` at the repo root